sklearn/preprocessing/data.py

```diff
@@ -2409,9 +2412,9 @@ class PowerTransformer(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
-    method : str, (default='box-cox')
-        The power transform method. Currently, 'box-cox' (Box-Cox transform)
-        is the only option available.
+    method : str, (default='yeo-johnson')
+        The power transform method. Available methods are 'box-cox' and
```

Comment:
We can maybe have a bullet point list for each method referring to the reference section.